### ISSUE TITLE
.model.json files break live sync

### ISSUE THREAD

**Issue by icanismar** [Iron-Stag-Games/Lync#21]
.model.json files are not being parsed correctly, and seemingly, they break live sync. 

Saving the same .model.json file produces a series of odd logs, which may be a lead:

Terminal:
```
Mapping JsonModel 'tree/ReplicatedStorage/Lycan/Tes' -> [source\platform\shared\TestModel.model.json]
Mapping JsonModel 'tree/ReplicatedStorage/Lyc' -> [source\platform\shared\TestModel.model.json]
Mapping JsonModel 'tree/ReplicatedStora' -> [source\platform\shared\TestModel.model.json]
```

Studio:
```
'ReplicatedStora' is not a valid Service name
  Stack Begin
  Script 'user_Lync.rbxm.Plugin', Line 441 - function buildPath
  Script 'user_Lync.rbxm.Plugin', Line 876
  Stack End
```

**To reproduce:**

1. Add a .model.json file to your project
2. Start live sync with debug ON
3. Modify the .model.json file 
4. On Studio, in the explorer, notice how a duplicate model appears under a different name (and parent in some cases)
5. Modifying the file again a few times eventually breaks live sync

**Comment by hoontee** [comment]
Thanks for the report. I'll have this fixed in a few hours.

**Comment by hoontee** [comment]
Fixed! Please update to [Alpha 15.3+](https://github.com/Iron-Stag-Games/Lync/releases) or [enable automatic updates](https://github.com/Iron-Stag-Games/Lync#configuration).
